ui/start_ui.py

```diff
@@ -48,6 +94,13 @@
     RAGTextFix,
 )
 from pyrecdp.primitives.document.reader import _default_file_readers
+from pyrecdp.core.cache_utils import RECDP_MODELS_CACHE
+
+import logging
+
+lib_list = ["httpcore", "httpx", "paramiko", "urllib3", "markdown_it", "matplotlib"]
+for lib in lib_list:
+    logging.getLogger(lib).setLevel(logging.ERROR)
```

Comment:
If this is to solve the DEBUG log problem, I think it can be solved after https://github.com/intel/llm-on-ray/pull/101 is merged.